shell/src/main/java/alluxio/cli/fsadmin/quorum/QuorumInfoCommand.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fsadmin.quorum;
+
+import alluxio.cli.fsadmin.command.AbstractFsAdminCommand;
+import alluxio.cli.fsadmin.command.Context;
+import alluxio.client.journal.JournalMasterClient;
+import alluxio.conf.AlluxioConfiguration;
+import alluxio.exception.ExceptionMessage;
+import alluxio.exception.status.InvalidArgumentException;
+import alluxio.grpc.GetQuorumInfoPResponse;
+import alluxio.grpc.JournalDomain;
+import alluxio.grpc.QuorumServerInfo;
+
+import com.google.common.annotations.VisibleForTesting;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Options;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+/**
+ * Command for querying journal quorum information.
+ */
+public class QuorumInfoCommand extends AbstractFsAdminCommand {
+  public static final String DOMAIN_OPTION_NAME = "domain";
+
+  public static final String OUTPUT_HEADER_DOMAIN = "Journal domain\t: %s";
+  public static final String OUTPUT_HEADER_QUORUM_SIZE = "Quorum size\t: %d";
+  public static final String OUTPUT_SERVER_INFO = "%s\t %s:%s";
+
+  /**
+   * @param context fsadmin command context
+   * @param alluxioConf Alluxio configuration
+   */
+  public QuorumInfoCommand(Context context, AlluxioConfiguration alluxioConf) {
+    super(context);
+  }
+
+  /**
+   * @return command's description
+   */
+  @VisibleForTesting
+  public static String description() {
+    return "Shows quorum information for embedded journal.";
+  }
+
+  @Override
+  public int run(CommandLine cl) throws IOException {
+    // Validate domain option is correct.
+    String domainVal = cl.getOptionValue(DOMAIN_OPTION_NAME);
+    try {
+      JournalDomain.valueOf(domainVal);
+    } catch (IllegalArgumentException e) {
+      throw new InvalidArgumentException(ExceptionMessage.INVALID_OPTION_VALUE
+          .getMessage(DOMAIN_OPTION_NAME, Arrays.toString(JournalDomain.values())));
+    }
+
+    JournalMasterClient jmClient = mMasterJournalMasterClient;
+
+    if (cl.hasOption(DOMAIN_OPTION_NAME)) {
```

Comment:
Is it necessary? Can we directly use the previous try clause to get the `domain`.